src/components/ApiReferences/_helpers.js

```diff
@@ -0,0 +1,6 @@
+const idPattern = /[^A-Za-z0-9\-]/g;
+export const nameToId = (name, version) => {
+  let slug = name.trim().replaceAll(" ", "-").replace(idPattern, "");
+  let suffix = version === "Clarity1" ? "" : `-${version}`.toLowerCase();
```

Comment:
The suffix is displayed in the url, for example:
```
/clarity/language-functions#element-at
```
To avoid broken links, this method generates the same slugs as docusaurus natively does. So with this new PR, all of the links will still work.

Because Clarity 2 introduces aliases (`element-at?` and `index-of`), I included the clarity version in the anchor, except for Clarity 1 methods.
So 
- `element-at` anchor will be `#element-at`
- `element-at?` anchor will be `#element-at-clarity2`

If clarity 3 introduces somethings like `element-at!` it would be `#element-at-clarity3`